### PR TITLE
Cloudinary

### PR DESCRIPTION
**Pull request by samar12-rad**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Fixes #(issue_no)

Replace `issue_no` with the issue number which is fixed in this PR

## Checklist

- [ ] Tests have been added or updated to cover the changes
- [ ] Documentation has been updated to reflect the changes
- [ ] Code follows the established coding style guidelines
- [ ] All tests are passing

### CODE REVIEW

**Comment by samar12-rad** [comment]
on hold

**Comment by nishant0708** [comment]
@samar12-rad  solve merge conficts and push latest code

**Comment by samar12-rad** [comment]
Check Now

**Comment by samar12-rad** [comment]
@nishant0708

**Comment by nishant0708** [comment]
@samar12-rad conflicts

**Comment by Kartikey2011yadav** [comment]
@nishant0708 conflicts resolved do review checks and merge

File: src/Reset/Reset.jsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
+import { Link, useLocation } from 'react-router-dom';
```
Comment (nishant0708) [inline src/Reset/Reset.jsx]: What is this? reset already exist!!

**Comment by nishant0708** [comment]
image

**Comment by samar12-rad** [comment]
@nishant0708 check

**Comment by samar12-rad** [comment]
@nishant0708 check

**Review by nishant0708 — APPROVED** [review]

**Pull request merged**